my-dsl.md

```diff
@@ -2,37 +2,71 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Selenium Commands - "Selenese"](http://www.seleniumhq.org/docs/02_selenium_ide.jsp#selenium-commands-selenese)
 
 # Domain
 _Describe the language's domain in five words._
 
+Automated tests for browser-based applications.
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+I assume that each Selenese command is mapped to some equivalent code in another 
+language.  For example, `open /downloads/` could be some shell command to open the 
+browser and navigate to that URL). So the commands are parsed and the
+matching code is run with the parameters entered.  Especially because the
+testing is done for applications in the browser, each Selenese command could be
+mapped to some Javascript code that is run when the Selenese command is invoked.
```

Comment:
At a more abstract level, I may describe the Computational Model as follows:

In the form of Selenium commands, a user expresses actions on a web page and there expected result and content. The test runner performs each of the described actions and checks whether the webpage is in the expected state. If it is not in an expected state, the runner adds errors, if not it continues testing by performing each of the described actions and assertions.